testdata/project-v3-addon/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -15,7 +15,6 @@ spec:
         - "--secure-listen-address=0.0.0.0:8443"
         - "--upstream=http://127.0.0.1:8080/"
         - "--logtostderr=true"
-        - "--v=10"
```

Comment:
WDYT about we change to 0, then we would have the same result but would still easier for users know how to debug that if/when required?

Reply:
I like that! I went back and forth between removing and setting it. I'll update to use 0 so that it's more clear that the flag is available if needed. Thank you.

Reply:
Done, apologies for the force pushes I had the wrong git config setup 🤦